server/service/src/sync/translations/master_list_line.rs

```diff
@@ -38,18 +41,26 @@ impl SyncTranslation for MasterListLineTranslation {
         _: &StorageConnection,
         sync_record: &SyncBufferRow,
     ) -> Result<PullTranslateResult, anyhow::Error> {
-        // TODO, check site ? (should never get delete records for this site, only transfer other half)
+        // TODO, check site ? (should never get delete records for this site,
+        // only transfer other half)
         Ok(PullTranslateResult::delete(MasterListLineRowDelete(
             sync_record.record_id.clone(),
         )))
     }
 
     fn try_translate_from_upsert_sync_record(
         &self,
-        _: &StorageConnection,
+        connection: &StorageConnection,
         sync_record: &SyncBufferRow,
     ) -> Result<PullTranslateResult, anyhow::Error> {
         let data = serde_json::from_str::<LegacyListMasterLineRow>(&sync_record.data)?;
+        let master_list =
+            MasterListRowRepository::new(connection).find_one_by_id(&data.item_master_ID)?;
+
+        if master_list.is_none() {
+            return Ok(PullTranslateResult::Ignored("Orphaned line".to_string()));
```

Comment:
Maybe saying "Missing master list" is more clear?

Nit: remove empty line...

Reply:
What empty line? 🤔

Reply:
the empty line just before `if master_list.is_none() {` The previous statement is only used in the if statement. Think its nicer to have them close together, with the extra line it looks like they don't belong together...